.github/workflows/full-bundle-tests.yaml

```diff
@@ -96,25 +92,40 @@ jobs:
           sudo sysctl fs.inotify.max_user_instances=1280
           sudo sysctl fs.inotify.max_user_watches=655360
 
-      - run: |
+      - name: Configure juju model
+        run: |
           sg microk8s -c "juju add-model kubeflow --config default-series=focal --config automatically-retry-hooks=true"
           sg microk8s -c "juju model-config"
           sg microk8s -c "juju status"
 
-      - run: |
-          # required for gecko driver
-          export XDG_RUNTIME_DIR="/run/user/$(id -u)"
-          export DBUS_SESSION_BUS_ADDRESS="unix:path=$XDG_RUNTIME_DIR/bus"
-          echo "$(id -u)"
-          loginctl enable-linger $USER
-          sudo apt-get install dbus-user-session -yqq
-          systemctl --user start dbus.service
+      # - name: Configure env for Gecko driver
+      #   run: |
+      #     # required for gecko driver
+      #     export XDG_RUNTIME_DIR="/run/user/$(id -u)"
+      #     export DBUS_SESSION_BUS_ADDRESS="unix:path=$XDG_RUNTIME_DIR/bus"
+      #     echo "$(id -u)"
+      #     loginctl enable-linger $USER
+      #     sudo apt-get install dbus-user-session -yqq
+      #     systemctl --user start dbus.service
 
       - name: Run bundle tests
         run: |
-          export BUNDLE_TEST_PATH=${{ inputs.bundle-test-path }}
+          # export BUNDLE_TEST_PATH=${{ inputs.bundle-test-path }}
+          eval "$(pyenv init -)"
+          export BUNDLE_TEST_PATH=tests-bundle/1.7
           export GH_TOKEN=${{ secrets.GITHUB_TOKEN }}
-          sg microk8s -c "tox -e full_bundle_tests -- ${{ inputs.bundle-source }}"
+          # sg microk8s -c "tox -e full_bundle_tests -- ${{ inputs.bundle-source }}"
+          sg microk8s -c "tox -e full_bundle_tests -- --file releases/1.7/stable/kubeflow/bundle.yaml"
+
+      - name: Checkout Kubeflow UATs
+        run: |
+          git clone https://github.com/canonical/charmed-kubeflow-uats.git ~/charmed-kubeflow-uats
+          cd ~/charmed-kubeflow-uats
+          git checkout main
+      - name: Run UATs
+        run: |
+          eval "$(pyenv init -)"
+          sg microk8s -c "tox -c ~/charmed-kubeflow-uats/ -e kubeflow -- --env params.env"
```

Comment:
Small reminder to remove the `--env params.env` part and the code that generates this file.

It was initially used to inject ENV Vars for the Proxy in all pods generated by the UATs